Tensile/Source/client/include/HardwareMonitor.hpp

```diff
@@ -143,6 +146,14 @@ namespace Tensile
 
             std::vector<uint32_t> m_fanMetrics;
             std::vector<int64_t>  m_fanValues;
+            // Below 3 vectors stores the individual values of GFx frequency, GPU power, GPU temperature during GEMM kernel execution.
+            // These vectors are implemented slightly different from previous clock/fan metric type implementation which uses add**Monitor functions to              // monitor and store multiple HW types information in the vectors for the same device. (ie)Existing metric implementation represent different            // HW type and uses ROCm API to get multiple HW type(like different sensor, different clock). Hence it uses add**Monitor functions to store              // in multiple vectors. each new HW type requires an invocation of ROCm API.
+            // but below vectors uses ROCm API (rsmi_dev_gpu_metrics_info_get) through single invocation gets all the HW type details hence it dont need
+            // exisitng type of implementation. if we need to store different HW type in the future, new additonal vector type or vector<vector>>would be            // appropriate.
```

Comment:
Please use spell checker for your code change.
  exisitng -> existing
  dont -> don't or do not
  additonal -> additional